packages/node_modules/@node-red/runtime/lib/nodes/flows/Flow.js

```diff
@@ -461,6 +461,10 @@ class Flow {
         }
         console.log("==================")
     }
+
+    getenv(name) {
```

Comment:
This is what `getSetting` does - we don't need `getenv` adding to the Flow class.